.github/workflows/release.yml

```
@@ -35,4 +35,7 @@ jobs:
           env:
             tagName: ${{ steps.getTag.outputs.tag }}
           run: |
-            ./dev/google-cloud split $GITHUB_REPOSITORY $tagName -t ${{ secrets.SPLIT_TOKEN }}
+            ./dev/google-cloud split $GITHUB_REPOSITORY $tagName \
+              -t ${{ secrets.SPLIT_TOKEN }} \
+              --packagist-username=${{ secrets.PACKAGIST_USERNAME }}
```

As this data is not sensitive, use an environment var instead of a secret
```suggestion
              --packagist-username=${{ vars.PACKAGIST_USERNAME }}
```